src/Microsoft.FeatureManagement/VariantFeatureManagerExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.FeatureManagement.FeatureFilters;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Extensions used to provide an overload for <see cref="IVariantFeatureManager.GetVariantAsync(string, ITargetingContext, CancellationToken)"/> that accepts a <see cref="TargetingContext"/> type directly.
```

Comment:
This class summary is too coupled to the method summary. It's unnecessary and would need to be updated if the method was updated or ever an other method was added.

```suggestion
    /// Extensions for <see cref="IVariantFeatureManager"/>.
```